velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -383,14 +383,16 @@ TEST_F(HdfsFileSystemTest, missingFileForWrite) {
 }
 
 TEST_F(HdfsFileSystemTest, writeDataFailures) {
-  const std::string errorMsg = "Write failure in HDFSWriteFile::append";
+  const std::string errorMsg =
```

Comment:
nit: this variable is not needed, just use the string directly in VELOX_ASSERT_THROW